src/modules/flow/netctl/netctl.json

```diff
@@ -0,0 +1,80 @@
+{
+  "$schema": "http://solettaproject.github.io/soletta/schemas/node-type-genspec.schema",
+  "name": "netctl",
+  "meta": {
+    "author": "Intel Corporation",
+    "license": "Apache-2.0",
+    "version": "1"
+  },
+  "types": [
+    {
+      "category": "netctl",
+      "description": "Send list of all network services attached to board.",
+      "methods": {
+        "open": "open_list_services",
+	"close": "close_list_services"
+      },
+      "name": "netctl/list-services",
+      "out_ports": [
+        {
+          "data_type": "string",
+          "description": "Name of network services supported by system.",
+          "name": "OUT"
+        }
+      ],
+      "private_data_type": "list_service_data"
+    },
+    {
```

Comment:
all in all, `netctl/connection` should be `netctl/service` and be close to https://github.com/solettaproject/soletta/blob/master/src/modules/flow/platform/platform.json#L573 (you can use `CONNECT` instead of `START` and `DISCONNECT` instead of `STOP`), but try to keep the overall usage, like `NAME` to set the service name OR use it from options. Also use a string port `OUT` as you did, but you may add helper `ONLINE` or `CONNECTED` boolean ports to notify in an easy way if that service is usable.